customizations.md

```diff
@@ -0,0 +1,12 @@
+# Customizations
+Based on your company, there are essential customizations that depend on your preferences and location. 
+
+
+### Timezone
+Set a timezone in your Looker Admin located at the top right corner. Scheduled Looks will be run in the time zone of your choosing. To set a database querying time zone, you’ll have to go into Connections on the sidebar. 
+
+### Local Currency
+In the [sales](sales.view.lkml) view, choose a  value_format_name based on your local currency or currency preference.
```

Comment:
Change to:

In the [model file](shopify.model.lkml#L7), specify a [currency format string](https://docs.looker.com/reference/model-params/named_value_format) for your local currency.